src/components/atoms/OcContextualHelper/OcContextualHelper.vue

```diff
@@ -84,6 +98,25 @@ export default {
   .info-more-link {
     font-size: var(--oc-font-size-small) !important;
   }
+  .info-header {
+    align-items: center;
+  }
+  .info-title {
+    font-size: 1.125rem;
```

Comment:
Do we have a font size theming variable for this?

Reply:
unfortunately no, but we use the same size in the OcButton.vue component